### PR TITLE
Fix a few offline issues found while testing 0.1.11.

### PR DESCRIPTION
**Pull request by kenzieschmoll**

### CODE REVIEW

File: packages/devtools_app/lib/src/timeline/html_timeline_screen.dart [hunk 1]
```diff
@@ -386,7 +386,10 @@ class HtmlTimelineScreen extends HtmlScreen {
             sectionSpacing;
   }
 
-  void _configureSplitter() {
+  void _configureSplitter({List<int> sizes}) {
+    if (sizes != null) {
```
Comment (jacob314) [inline packages/devtools_app/lib/src/timeline/html_timeline_screen.dart]: move the if into the assert.

Comment (kenzieschmoll) [reply packages/devtools_app/lib/src/timeline/html_timeline_screen.dart]: done

File: packages/devtools_app/lib/src/timeline/html_timeline_screen.dart [hunk 2]
```diff
@@ -387,9 +387,7 @@ class HtmlTimelineScreen extends HtmlScreen {
   }
 
   void _configureSplitter({List<int> sizes}) {
-    if (sizes != null) {
-      assert(sizes.length == 2);
-    }
+    assert(sizes?.length ?? 2 == 2);
```
Comment (jacob314) [inline packages/devtools_app/lib/src/timeline/html_timeline_screen.dart]: nit: probably simpler to just write
assert(size == null || sizes.length == 2)

Comment (kenzieschmoll) [reply packages/devtools_app/lib/src/timeline/html_timeline_screen.dart]: that is simpler. Done.

**Pull request merged**